run/identify-sha.sh

```diff
@@ -0,0 +1,14 @@
+# Copyright 2018 The WPT Dashboard Project. All rights reserved.
+
+# Use of this source code is governed by a BSD-style license
+# that can be found in the LICENSE file.
+
+# Run this script in cron, and have it write to the
+# /var/www/html folder a file called sha
+#
+# You should have Options +Includes, mod_include and XBitHack on in
```

Comment:
Can you make this part of some checked-in config?

Reply:
Sure, I'll add this.